trl/trainer/ppov2_trainer.py

```diff
@@ -0,0 +1,799 @@
+import gc
+import os
+import time
+from collections import OrderedDict, defaultdict
+from dataclasses import dataclass
+from typing import Dict, List, Literal, Optional, Tuple, Union
+
+import numpy as np
+import pandas as pd
+import torch
+import torch.nn as nn
+import torch.nn.functional as F
+from accelerate import Accelerator
+from accelerate.state import AcceleratorState
+from accelerate.utils import broadcast, gather_object
+from datasets import Dataset
+from torch.utils.data import DataLoader
+from transformers import (
+    DataCollatorWithPadding,
+    GenerationConfig,
+    PreTrainedTokenizer,
+    Trainer,
+    TrainerCallback,
+    TrainerControl,
+    TrainerState,
+    TrainingArguments,
+)
+from transformers.integrations import get_reporting_integration_callbacks
+from transformers.trainer_callback import CallbackHandler, DefaultFlowCallback
+
+from trl.models.utils import unwrap_model_for_generation
+from trl.trainer.utils import exact_div, print_rich_table
+
+
+INVALID_LOGPROB = 1.0
+
+
+@dataclass
+class PPOConfig(TrainingArguments):
```

Comment:
For consistency with our other trainers, can we move this to `ppov2_config.py` ?